src/components/billingRecord/IFXBillingRecordListDecimal.vue

```diff
@@ -968,6 +974,15 @@ export default {
               <span style="white-space: nowrap">
                 {{ item.endDate | humanDatetime }}
               </span>
+            </template>
+            <template v-slot:item.productUsage="{ item }">
+              <span v-if="item.productUsageLinkText" style="white-space: nowrap">
```

Comment:
Same thing here; you can use the Vuetify `text-no-wrap` helper class instead of an inline style.